### DISCUSSION TITLE
Faster upgrade process?

### DISCUSSION THREAD

**Discussion by illuminos** [jesseduffield/lazygit#2287] [Ideas]
Hello.

I've used lazy git every day for the last 6 months and its one of my favorite terminal clients now andI recommend it to everyone I meet. Million thanks for this !

I had an idea about the lazygit upgrading process. Whenever I look at the bottom right corner, I see the version number but I'm always curious if there is a newer version available. So I always click the "ask question" link to go to the github repo and check the tags to see if a newer version is published. Is it possible to display an upgrade-available indicator - maybe like an asterisk next to the version number ? And when you click on it, it will upgrade lazygit and put you back on the home screen. It might be a hard challenge to know where they installed it from (is homebrew the only place where lazygit is available?)

I don't know if this idea is even possible to implement or if someone is already looking into it. Let me know if this is a good first issue to tackle.

**Comment by mark2185** [comment]
You could press `u` on the `Status` panel (under number `1`), or leave the auto update feature enabled.
```
update:
  method: prompt # can be: prompt | background | never
  days: 14 # how often an update is checked for
```

More info can be found [here](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md).

**Reply by illuminos** [reply]
That's amazing. Didn't know that. Thank you !
